### ISSUE TITLE
Groups

### ISSUE THREAD

**Issue by curly60e** [LN-Juggernaut/juggernaut-desktop#4]
I was thinking, how about adding groups, how difficult could be?
I know this is new, but if all want to move from Signal, Telegram, and all other chat to this "Groups" should be cool to add.

**Comment by johncantrell97** [comment]
Yeah, this is planned feature.  It's a little more complicated to implement without a centralized party to organize the group and its members.
